# 15 Crack gap

Does the correlation between the crack's distance from its pre-2020 median and the next five-print RWTC return keep its sign after 2019?

Holdout starts 2020-01-01. Verdict: failed.

CRACK_321 minus its train median. Train median 16.739 dollars per barrel. CRACK_321 is ((2 * RBOB_F1 + HO_F1) / 3) * 42 minus RCLC1. A date is absent when any input is absent. The futures workbooks end before the spot series.

In [1]:
import json
from pathlib import Path
studies = json.loads(Path('data/studies.json').read_text())
study = next(item for item in studies['studies'] if item['slug'] == 'crack-gap')
print(study['verdict'])
print(study['verdict_line'])
print(study['table']['caption'])
for row in study['table']['rows']:
    print(' | '.join(row))


failed
Train correlation of the crack gap with the next five-print RWTC return is 0.036. Holdout correlation is -0.030. The holdout flipped the train sign.
Crack minus the frozen train median, versus the next five RWTC prints
Train | 3371 | 0.036
Holdout | 1066 | -0.030


## Method

The bar is sign agreement of the correlation. A rich crack is not given a second rule after the result is in.

## Result

Train correlation of the crack gap with the next five-print RWTC return is 0.036. Holdout correlation is -0.030. The holdout flipped the train sign.